zzre/game/systems/gameflow/Teleporter.cs

```diff
@@ -88,6 +88,7 @@ private void HandleTeleport(in messages.Teleport message)
         if (!IsEnabled)
             return;
 
+        World.Publish(new messages.SpawnSample($"resources/audio/sfx/specials/_s009.wav", AsEntity: ui.World.CreateEntity()));
```

Comment:
You generally don't need to assign `AsEntity` as you have already done with the other sounds, except for this sample right here as it is a bit more complicated. 
More specifically:
- It is spawned with `Volume: 0f`
- During `State.Leaving` the volume is increased using `(LeavingDuration - timeLeft) / 3` (untested)
- Before starting `_s010` it is cut off and destroyed